### PR TITLE
EREGCSC-1500 Add view resource button to pages

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves #
EREGCSC-1500

**Description-**

Adds the view resources button to all pages except for  the resources ppage
**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

1. steps to view and verify change

Go the following pages and see if the "View Resources" Button appears in the toolbar
 "/" home page
"/about"
"/search"

Go to "/resources" and verify that the button is not there

### CODE REVIEW

File: solution/backend/regulations/templates/regulations/partials/header/reg-title.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {% load static %}
 
-{% if part %}
+
     <div class="reg-title">
```
Comment (PhilR8) [inline solution/backend/regulations/templates/regulations/partials/header/reg-title.html:4]: Having `reg-title` on every page is a fine solution to this, but it adds a gray left border to the header:

Screen Shot 2022-09-15 at 09 12 47

That border is generally meant to separate the site title ("Medicaid & CHIP eRegulations") and the reg title on the Part TOC/Reader view pages (ex: "42 CFR Part 433").  It's there's no reg title, the border should be hidden.

My vote would be to hide it on the homepage/about page/search results page, but you might ask the designers if it's okay to leave it here.

File: solution/ui/regulations/css/scss/partials/_header.scss [hunk 2]
```diff
@@ -308,7 +308,6 @@ header {
         }
 
         .reg-title:after {
-            content: "Search Results";
```
Comment (PhilR8) [inline solution/ui/regulations/css/scss/partials/_header.scss:311]: was this left over from a long forgotten version of the header?

**Review by PhilR8 — CHANGES_REQUESTED** [review]
There are some python linter issues that should be corrected, as well as a small styling issue that could go either way.

File: solution/ui/regulations/css/scss/partials/_header.scss (hunk 2 above)
Comment (thwalker6) [reply solution/ui/regulations/css/scss/partials/_header.scss:311]: Thats what I was thinking.  If you go to that page it actually doesnt appear anywhere but with my changes it did appear.  Befoer when we did the if part on the django template, the .reg-title was hidden.

Comment (PhilR8) [reply solution/ui/regulations/css/scss/partials/_header.scss:311]: okay nice.  deleting stale code is always good!

File: solution/backend/regulations/templates/regulations/partials/header/reg-title.html (hunk 1 above)
Comment (thwalker6) [reply solution/backend/regulations/templates/regulations/partials/header/reg-title.html]: Added some django/css logic to make it appear if part appears

**Review by PhilR8 — APPROVED** [review]
LGTM!

**Pull request merged**